pyomo/contrib/sensitivity_toolbox/tests/test_sens.py

```diff
@@ -367,6 +369,307 @@ def test_parameter_example(self):
             # Check each element of the 'correct' dictionary against the returned 
             # dictionary to 3 decimal places
             self.assertAlmostEqual(d[k],d_correct[k],3)
+    
+    
+    # Test kaug
+    # Perform the same tests as ipopt_sens 
+    # test arguments
+    @unittest.skipIf(not opt_kaug.available(False), "k_aug is not available")
+    @unittest.skipIf(not opt_dotsens.available(False), "dot_sens is not available")
+    def test_kaug_bad_arg_kaug(self):
+        m = ConcreteModel()
+        m.t = ContinuousSet(bounds=(0,1))
+
+        m.a = Param(initialize=1, mutable=True)
+        m.b = Param(initialize=2, mutable=True)
+        m.c = Param(initialize=3, mutable=False)
+
+        m.x = Var(m.t)
+
+        list_one = [m.a,m.b]
+        list_two = [m.a,m.b,m.c]
+        list_three = [m.a, m.x]
+        list_four = [m.a,m.c]
+
+        # verify ValueError thrown when param and perturb list are different
+        # lengths
+        try:
+            Result = kaug(m,list_one,list_two)
+            self.fail("Expected ValueError: for different length lists")
+        except ValueError:
+            pass
```

Comment:
Could you update these tests to use the `assertRaises` syntax instead of `try/except`? The syntax should look like:
```
with self.assertRaises(ValueError):
    Result = kaug(m, list_one, list_two)
```
I don't think the original developer of `sens.py` was aware of this syntax when she wrote the tests but this is the standard we've settled on for the rest of Pyomo.

Reply:
I edited it.